bench/bm_msg_forward.cc

```diff
@@ -0,0 +1,89 @@
+#include <chrono>
+#include <iostream>
+#include <thread>
+
+#include <gnuradio/blocks/msg_forward.h>
+#include <gnuradio/flowgraph.h>
+#include <gnuradio/realtime.h>
+#include <gnuradio/runtime.h>
+#include <pmtf/base.hpp>
+
+#include <iostream>
```

Comment:
`iostream` is included twice. It'd be interesting to replace `iostream` with `fmt`. e.g. use `fmt::print`.